front/public_html/js/data.js

```diff
@@ -17,7 +17,7 @@ function getIslands() {
  */
 function getSeasons() {
     return new Promise((resolve, reject) => {
-        getDataFromEntryPoint("/temporadas", resolve);
+        getDataFromEntryPoint("http://islasgeci.org:8510/api-datos/temporadas", resolve);
```

Comment:
Tiene finta de número mágico, al menos sería posible encapsular en una variable
```suggestion
        cons pathData = "http://islasgeci.org:8510/api-datos/temporadas";
        getDataFromEntryPoint(pathData, resolve);
```
Los cambios en este archivo muestran que no está de más crear una constante. Tal vez debería de ser: `let path_data = "http://islasgeci.org:8510/api-datos/temporadas";`

Reply:
@nepito: Atendí esto mediante unas variables globales al inicio del _script_. Sin pruebas, es difícil saber si esta refactorización funciona correctamente o no, así que me persigné antes de consignarlo y después de pusharlo. 🤞